src/content/popup/names.ts

```diff
@@ -22,143 +17,26 @@ export function renderNamesEntries({
   more: boolean;
   options: ShowPopupOptions;
 }): HTMLElement {
-  const namesTable = html('div', { class: 'name-table entry-data' });
+  const containerElement = html('div', {
+    /* Make sure it's possible to scroll all the way to the bottom of each kanji
+     * table. */
+    style:
+      'scroll-snap-align: start; scroll-margin-bottom: var(--expand-button-allowance);',
+  });
 
-  if (options.meta) {
-    const metadata = renderMetadata({
+  render(
+    h(NameTable, {
+      entries,
+      matchLen,
+      more,
       fxData: options.fxData,
       preferredUnits: options.preferredUnits,
-      isCombinedResult: true,
-      matchLen,
+      copyState: options.copyState,
+      onStartCopy: options.onStartCopy,
       meta: options.meta,
-    });
-    if (metadata) {
-      namesTable.append(metadata);
-    }
-  }
```

Comment:
When do we show metadata in the names tab? Can't think of a scenario off the top of my head.